### DISCUSSION TITLE
Arrow Right komponent

### DISCUSSION THREAD

**Discussion by ivnai** [fremtind/jokul#2713] [Annet]
.. blir til Checkmark i kompakt variant.
![Screenshot 2022-02-08 at 11 55 44](https://user-images.githubusercontent.com/93194579/152974172-8938d66e-b744-4ba5-8d64-efc70bfe3bd8.png)
![Screenshot 2022-02-08 at 11 55 59](https://user-images.githubusercontent.com/93194579/152974353-e0412aff-7c87-448a-9e87-592ca41f6948.png)

**Comment by wkillerud** [comment]
På Icons-siden i biblioteket er det bare Checkmark, Close, og Plus som har Compact-variant akkurat nå. 

[I kode](https://jokul.fremtind.no/komponenter/icons) er det både `small`, `medium` og `large`-varianter, men de er basert på fontstørrelse. Det betyr at de får forskjellig stroke, ulikt Figma 🤔

Jeg ser at jeg lukket #1793 i fjor høst. Det var for tidlig, innser jeg nå 😄 Hva tenker du vi bør gjøre @Slaine2?
